docker-compose.yml

```diff
@@ -187,169 +157,25 @@ services:
-      - MERCHANT_FILE=simple_competition_logic/MerchantApp.py
-      - MERCHANT_PORT=5002
-      - MERCHANT_TOKEN=mNq3VfVIs5gNES1EB6MsKhiNCL7h32zpPqsvWNDPStmhNnvYtIRE0BbCViuYktOA
-    # volumes:
-    #   - ./pricewars-merchant/simple_competition_logic:/merchant
-    depends_on:
-      - producer
-      - marketplace
-    links:
-      - producer
-      - marketplace
-
-  merchant-sample-cheapest:
-    build: ./merchant
-    image: pricewars/merchant
     ports:
       - "5003:5003"
-    environment:
-      - PRICEWARS_MARKETPLACE_URL=marketplace:8080
-      - PRICEWARS_PRODUCER_URL=producer:3050
-      - PRICEWARS_KAFKA_REVERSE_PROXY_URL=kafka-reverse-proxy:8001
-      - MERCHANT_FILE=sample_merchant/CheapestMerchantApp.py
-      - MERCHANT_PORT=5003
-      - MERCHANT_TOKEN=xJvs838ArhvEiBDsvwBWlE3jsatH3uU0DQsLT1VLiSZSdmKZjV6Wj9GcjAtm5OmI
-    # volumes:
-    #   - ./pricewars-merchant/sample_merchant:/merchant
-    depends_on:
-      - producer
-      - marketplace
-    links:
-      - producer
-      - marketplace
-
-  merchant-sample-second-cheapest:
-    build: ./merchant
-    image: pricewars/merchant
-    ports:
-      - "5004:5004"
-    environment:
-      - PRICEWARS_MARKETPLACE_URL=marketplace:8080
-      - PRICEWARS_PRODUCER_URL=producer:3050
-      - PRICEWARS_KAFKA_REVERSE_PROXY_URL=kafka-reverse-proxy:8001
-      - MERCHANT_FILE=sample_merchant/SecondCheapestMerchantApp.py
-      - MERCHANT_PORT=5004
-      - MERCHANT_TOKEN=Qrlo6DEkG9SXNIjCByXklvpjbkbQC4scveS2rfm8oEfCWPNhOuSAqIsS4UtGYmsU
-    # volumes:
-    #   - ./pricewars-merchant/sample_merchant:/merchant
-    depends_on:
-      - producer
-      - marketplace
-    links:
-      - producer
-      - marketplace
-
-  merchant-sample-random-third:
-    build: ./merchant
-    image: pricewars/merchant
-    ports:
-      - "5005:5005"
-    environment:
-      - PRICEWARS_MARKETPLACE_URL=marketplace:8080
-      - PRICEWARS_PRODUCER_URL=producer:3050
-      - PRICEWARS_KAFKA_REVERSE_PROXY_URL=kafka-reverse-proxy:8001
-      - MERCHANT_FILE=sample_merchant/RandomThirdMerchantApp.py
-      - MERCHANT_PORT=5005
-      - MERCHANT_TOKEN=oXbPx3EEZw4vpBb3QmITBIu86PA48oWwN8LKAZYjXcfNoHQF4ihF1cO62YAANg6k
-    # volumes:
-    #   - ./pricewars-merchant/sample_merchant:/merchant
-    depends_on:
-      - producer
-      - marketplace
-    links:
-      - producer
-      - marketplace
-
-  merchant-sample-two-bound:
-    build: ./merchant
-    image: pricewars/merchant
-    ports:
-      - "5006:5006"
-    environment:
-      - PRICEWARS_MARKETPLACE_URL=marketplace:8080
-      - PRICEWARS_PRODUCER_URL=producer:3050
-      - PRICEWARS_KAFKA_REVERSE_PROXY_URL=kafka-reverse-proxy:8001
-      - MERCHANT_FILE=sample_merchant/TwoBoundMerchantApp.py
-      - MERCHANT_PORT=5006
-      - MERCHANT_TOKEN=EGkdMWbvFuVy9w7ReCExgOiyoxr6RyFfF0JbXuXmp4ly1sRf5fdl4MjsKbylbERq
-    # volumes:
-    #   - ./pricewars-merchant/sample_merchant:/merchant
+    command: python3 merchant_cheapest.py --port 5003
+    volumes:
```

Comment:
As this will be the default merchant from now on, I do not really like to merge it with the current name.

Could you rename the merchant to something simple as `merchant.py` or `bootstrap_merchant.py` and denote within the source that currently only the `cheapest strategy` is supported?